temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -205,3 +206,26 @@ message DeleteNamespaceResponse {
     // The async operation.
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
+
+message GetRegionsRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100. 
+    int32 page_size = 1;
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
```

Comment:
Where do I get page token from? It's not on the response.

Reply:
Thanks fixed.